src/AzOps.psm1

```diff
@@ -365,14 +367,14 @@ class AzOpsScope {
             $mgId = ((($this.scope -split $this.regex_managementgroupExtract) -split '/') | Where-Object { $_ } | Select-Object -First 1)
 
             if ($mgId) {
-                Write-AzOpsLog -Level Debug -Topic "AzOpsScope" -Message "Querying AzOpsAzManagementGroup for Id: $mgId"
-                $mgDisplayName = ($global:AzOpsAzManagementGroup | Where-Object { $_.Name -eq $mgId }).DisplayName
+                Write-AzOpsLog -Level Debug -Topic "pwsh" -Message "Querying Global variable for AzOpsAzManagementGroup"
```

Comment:
I'm not sure we want the topic back to [pwsh]?

I've also just gone through the code to set every instance of $global to use lowercase, but I think you've accidentaly reverted this during merge conflict resolution?

Reply:
Thanks! I’ll change it back - most likely due to merge conflict.